src/Compiler/Service/ServiceLexing.fsi

```diff
@@ -6,7 +6,6 @@ open System
 open System.Threading
 open FSharp.Compiler
 open FSharp.Compiler.Text
-
```

Comment:
Would probably be good to keep an empty line between opens and the directive?

Reply:
Looks like this may be fixed in latest preview https://fsprojects.github.io/fantomas-tools/#/fantomas/preview?data=N4KABGBEDGD2AmBTSAuKA7AhgW0QZwAdNpEwAxAZQAtMAnAgOgGFZsCBLAG0VoYBVYAa0Tp2AL0wAXdrHQAdWnPmL0sAiLAUAnnkmJsCpWo3bd%2B-lVqJM8dugDmh1evTlqdRizZceT468oaemZWDm5ePkQAD0knJwBiVQB3Olc5SABWAHZ0uJUAekKwACVEAis8EUk8MBE4JHgwOwAzWFpsKRlXVtowSSpSEXgAWlhm4c47Ujh0aXQAV07ZMDGwbii7RxVJLXUwAEEACgBKMABeMGhOTDwaoagAGnAoSsk5%2BzxUMABtZ4hQCCAqAAEh26i%2BkDskkgTyBEEgwIAbphOPNkGhfnDAQAGWFYqAASXQSFmFHEyDxWIALH9AQBdWkAX0pYABcIRYPRUChMNp8KRKLRX0x%2BIAjCygZAALKYKIAGSmcpE9n6vPxYFFACZsXywAygczaWzJaDdlzIEMAPLNBXoRAUHbcNXsgWorkirGaiWAyAAUWJ1ttFN18M4zUguv1gMNQONPtN4LQkAARrBYJxnSbkW7hSGwABmb3womVWiSMh2FEAOUQSUmdszcMktDRkaZLLj-M5ENT6cb-OzQoxeapRagFH07DgnFk%2B0k-vggam-cBzRRlTbBo7uo5Zp7aYz3oRg-deYyY8gFCIJAAQogeogAAp0HCIPS0FcQZutrFRiAxwFOxBbsk17Q8d1dIcfjzAA2C8r2IRA7wfOVYCSHhoBuRAiURWBMOkWRPz6FtEE3aNtyxXdEygMCV2PQVT3VLJ4OvJD7zaRAAFUCHUWhMMqHC8KWdAiLXTgN1-dsjQgkCaIPOjIMY-EAA4WMQ5COKYa5bhYdBdBbaBJDaUT11IySt2kyiE3NWij0U3N1QATjU292KsKV9GTHgTPEsy4T-MAAP%2BGS91A%2BS7JPByxVxPNL1YjSrBYGcRLHMSJP8qTYxC6iU3CiDIuHdVRXFWKEJIfZmnfLwOiI78-KBAKgtZbKbLyqyCugoqvVK%2BK3PtSdp0I1LTLI-8KJdWTcr7CKGKirFRULHrEIq98J2wKd0yGvM6tGwLxpNSbbPy2bCrFUdYqJElJEtdA%2BFoLQAHV2FVYbfN2pqgKo1rpuOnNTvm88lvK2hYHmYkABFEEmdb31qkj3v2%2BNJp5Ga-s6sU4NimUogJZo%2BAGdBfV86g2kkJ74BekcdXM8jLIm0LuVmBSOo9OFRWYrHZSJZp2CiS1eKkNpfSicp8DwLoiIyamMosrKrORpnUag1mgVFVTOaiUo4FoeBycp9UqWlhrMsAlqIRR37lbzUVnI1rW2ngKt5mwLzaGtAk9GwT4x1FBG6YOhnIGweZOGketEDINoOjeHg%2BD3JWlM9GL1Uge2dalEOw6mSP2ikWGL2gIJiHfAB9JJ2ApqgIxpsb-aRwOLfak70c9EqU%2Bx-ZaFoTAtEtWgFV0PWq7HQ2-bl%2Bmcsbl0WbzTVuvb2VO%2B73v%2B-YXQnZdnh3c973rbH035cD4PQ64bOo7z2P48txO4U1RaU6Xnu%2B4HyQM5P8Oc%2Bj-PYsLl9DJ4MuFdVT72CofSeitr5zVvudBeUQABqDEbx2FsA4IeRFlJG3pCbUBE9zRTyzM3FWgJNSA1gWQMGhkuhIOJJsNBI9MEQEaojLsDcIFNzRkQiAmpMawPBrASQABxN8wtRa3C6HQs8DC9TYOamA764F2FW3VJqDm7dM6nztDeGc0BBA3m7jot8eAboUFfElZ2KU8xpXqlg2WB9cH7h%2Boom%2BQJNTqxTjWOsUw7ySHQiIOO6hIY81EARdA%2BxiQeU3rQXe6orEgNkfYsKjjp6ENnrbFOABpRAZRcb4xEESExuAgw%2BXSsbWxOCA7gOhAnKBQJ8zJ3xNKWURN1p4CoBIg2UimF1xYTlI6TiamAnzG3BpZIHDcE7vYZ2VQHqIGTFKBAwYYkjRrntbpwFA59OSRwvM%2BZ54NP2JMew6ByHoEobIMkRypDzCsAIS6VRhLFOsYwmRn1rIOIUVspR%2BJ8z332aHHgWBpCIkQKhBwETXaBLsM9Lo0T8SxJWR9M2iSPkEO2eqfMMCGlv2kEGOUOBkzwEwFpWA4sHAePDo8uJrzDptU%2Bc4wZpCGngzXpgZM3BmlryoNoWYspKUIuYes3ptLUVfKxPmHhDTMnZOJFUIkejMCnOHpY5ZMtabjwqfI5mKT0WqIaVohVgggx4E7qDYkNYzDwGxRoxAIiKji1kLCrEO1%2BVrK%2Bu8rVaLvluL1XQBKiBmV4GgLQdg61AWIHgJxUQshIZXBfCEvlqra7qvrkKpJIr6UQHzGkhpxRMCXJBoIBNpS1V2I1e66p-04SG3gs2Kcr8FlFpsdGZ4dI8SQjwGQcWXxYmMiAA